conf/modules.config

```diff
@@ -53,9 +53,17 @@ process {
         ]
     }
 
+    withName: "CELLPOSE" {
+        containerOptions = '--user root'
```

Comment:
We get that Matplotlib message about creating a temporary directory, but I thought that was more of a warning and then we get the error `RuntimeError: cannot cache function 'map_coordinates': no locator available for file '/usr/local/lib/python3.8/site-packages/cellpose/dynamics.py'`
Here's the full trace:
```
Command error:
  Matplotlib created a temporary config/cache directory at /tmp/matplotlib-9erjb_bd because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
  Traceback (most recent call last):
    File "/usr/local/bin/cellpose", line 5, in <module>
      from cellpose.__main__ import main
    File "/usr/local/lib/python3.8/site-packages/cellpose/__main__.py", line 7, in <module>
      from cellpose import utils, models, io, core
    File "/usr/local/lib/python3.8/site-packages/cellpose/utils.py", line 13, in <module>
      from . import metrics
    File "/usr/local/lib/python3.8/site-packages/cellpose/metrics.py", line 2, in <module>
      from . import utils, dynamics
    File "/usr/local/lib/python3.8/site-packages/cellpose/dynamics.py", line 332, in <module>
      def map_coordinates(I, yc, xc, Y):
    File "/usr/local/lib/python3.8/site-packages/numba/core/decorators.py", line 212, in wrapper
      disp.enable_caching()
    File "/usr/local/lib/python3.8/site-packages/numba/core/dispatcher.py", line 863, in enable_caching
      self._cache = FunctionCache(self.py_func)
    File "/usr/local/lib/python3.8/site-packages/numba/core/caching.py", line 601, in __init__
      self._impl = self._impl_class(py_func)
    File "/usr/local/lib/python3.8/site-packages/numba/core/caching.py", line 337, in __init__
      raise RuntimeError("cannot cache function %r: no locator available "
  RuntimeError: cannot cache function 'map_coordinates': no locator available for file '/usr/local/lib/python3.8/site-packages/cellpose/dynamics.py'
```
I'll see if I can find a workaround or fix.

Reply:
Try adding these lines to the cellpose script block just before the actual `cellpose` command:
```bash
export HOME=$(pwd)
export NUMBA_CACHE_DIR=$(pwd)
```
The current working directory will be the workdir, which is exactly where these sorts of temp/cache files should go.

If this works, we can add a patch file for cellpose while we work on fixing the module itself. (run `nf-core modules patch cellpose`)

Reply:
I made a tiny syntax change and this seems to work great.
I'll mark this as resolved, but please open another comment if I've got the syntax wrong.